spec/controllers/tag_wranglers_controller_spec.rb

```diff
@@ -16,6 +87,23 @@
         end.to raise_exception(ActiveRecord::RecordNotFound)
       end
     end
+
+    it_behaves_like "denies access to unauthorized admins", :get, :show, params: { id: -1 }
```

Comment:
Could this also use a valid id? Same question for the other `it_behaves_like "denies access to unauthorized admins"` lines.